migrations/2_deploy_contracts.js

```diff
@@ -7,7 +7,7 @@ const FIFSRegistrar = artifacts.require('./FIFSRegistrar.sol');
 // TODO: align the contract name with the source code file name.
 const Registrar = artifacts.require('./Registrar.sol');
 const web3 = new (require('web3'))();
-const namehash = require('eth-ens-namehash');
+const namehash = require('../node_modules/eth-ens-namehash');
```

Comment:
It's currently an known issue for truffle having trouble to do it the regular way. But would be resolved in the next release.